File: 1-easy/problems/1217-minimum-cost-to-move-chips-to-the-same-position/solution.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int even = 0;
        for (int pos : position) {
            even += pos % 2 == 0;
        }
        return min(even, (int)position.size() - even);
    }
};

File: 1-easy/problems/1217-minimum-cost-to-move-chips-to-the-same-position/solution.ts
function minCostToMoveChips(position: number[]): number {
  let even = 0;
  for (const pos of position) {
    even += pos % 2 === 0 ? 1 : 0;
  }
  return Math.min(even, position.length - even);
}

export { minCostToMoveChips };
